services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -9339,6 +9350,24 @@ components:
       - WALLET_SHARED
       type: string
       description: An enumeration.
+    OsparcCreditsAggregatedByServiceGet:
+      title: OsparcCreditsAggregatedByServiceGet
+      required:
+      - osparc_credits
+      - service_key
+      - running_time_in_hours
+      type: object
+      properties:
+        osparc_credits:
```

Comment:
in theory these should all be camelcase in the web-api , i.e. `osparcCredits` , `serviceKey`, `runningTimeInHours`